### PR TITLE
Add use guidelines

### PR DESCRIPTION
**Pull request by znicholls**
Adds usage guidelines to OpenSCM

- [x] Implementation finished (N/A)
- [x] Tests added (N/A)
- [x] Documentation added (N/A)
- [x] Example added (in the documentation, to an existing notebook, or in a new notebook) (N/A)
- [x] Description in ``CHANGELOG.rst`` added (single line such as: ``(`#XX <https://github.com/openclimatedata/openscm/pull/XX>`_) Added feature which does something``)

### CODE REVIEW

**Comment by rgieseke** [comment]
Good idea to have some "expectation setting" in the Readme/Docs!

My 2-cent:

AGPL is not a "permissive" license in the usual usage in the OS community but a "copyleft" license i think (and MIT/BSD are permissive).

I strongly dislike co-authorships suggestions in other projects so would refrain here, too (though i haven't been much involved recently i probably should count much on this issue) - i'd think it's fine to notice that you're open to collaboration, but the license allows anyone to use it according to the license.

In OpenSCM's case it's also the case that it builds on other people's work heavily, so it's even more difficult whom to include. For PRIMAP-hist we went with

```
 When using this dataset, or one of its updates, please cite the DOI of the precise version of the dataset used and also the data description article to which this dataset is supplement. Please consider also citing relevant original sources when using the PRIMAP-hist dataset. References to the original sources can be found in the mansucript and on the dataset download page. 
```

So something like this might be appropriate for the SCMs shipping with OpenSCM in the future. I dislike lots of things about IIASA's SSP databases but i think them providing clear citation instructions and downloadable citation data is pretty great!

**Comment by swillner** [comment]
I'm also not too much a fan of such notices. Maybe a short notice suffices that "we are open about collaboration"? I suppose, a nice `CITATION` file (with a specific formatted citation to copy-and-paste) would be of more value.

**Comment by znicholls** [comment]
Ok. So how about we go for:

- notice in similar vein to PRIMAP-HIST (i.e. please consider citations of us and original source, open to collaboration)
- `CITATIONS` file, in which we have citations for each model plus OpenSCM

?

**Comment by swillner** [comment]
Sorry, I missed your understanding of the ``CITATION`` file :/ It's supposed to include the way we would like OpenSCM to be cited (hence the singular).
Nevertheless, we should add a `cite` method to the adapter including those model citation.

**Comment by swillner** [comment]
... I like the new encouraging colaboration paragraph! Can you sort the maintainers alphabetically again, please?

**Comment by znicholls** [comment]
> Nevertheless, we should add a `cite` method to the adapter including those model citation.

Should we also provide a file which has all the citations in one (pulling citations from all over the place is one of the things I hate most when writing papers)?

**Comment by znicholls** [comment]
> Can you sort the maintainers alphabetically again, please?

Done

**Comment by swillner** [comment]
> > Nevertheless, we should add a `cite` method to the adapter including those model citation.
> 
> Should we also provide a file which has all the citations in one (pulling citations from all over the place is one of the things I hate most when writing papers)?

Maybe a function that collects all `cite` values from all available adapters?

**Comment by znicholls** [comment]
> Maybe a function that collects all `cite` values from all available adapters?

Just feels unnecessarily complex, why not just have a file?

**Comment by rgieseke** [comment]
I think R has some standards to get a `citation` but i'm not aware of any default ways in Python. I think having a file (or just in the Readme) should be easier.

**Comment by swillner** [comment]
Ok, then maybe rename `CITATIONS` to `CITATION` and add a way to cite OpenSCM on top with an explanatory text and then give the other citations?

**Comment by znicholls** [comment]
> Ok, then maybe rename `CITATIONS` to `CITATION` and add a way to cite OpenSCM on top with an explanatory text and then give the other citations?

Like so? Given we don't have a citation yet this is the best we can do right?

File: README.rst [hunk 1]
```diff
@@ -28,6 +28,37 @@ Python 3.
...
 
+Use guidelines
+--------------
+
+We encourage use of OpenSCM as much as possible and are open to collaboration.
+If you plan to publish using OpenSCM, please be respectful of the work and the `Maintainers`_' willingness to open source their code.
+
+In particular, when using OpenSCM, please cite the DOI of the precise version of the package used and consider citing our package description paper [when it's written, which it's not yet :)].
+As appropriate, please consider also citing the wrappers and models that OpenSCM relies.
+References to the wrappers and original models can be found in the documentation and are available in bibtex format in the ``CITATIONS`` file.
```
Comment (swillner) [inline README.rst]:
```suggestion
A way to cite OpenSCM alongside the references to the wrappers and original models can be found in the documentation and are available in bibtex format in the ``CITATION`` file.
```

File: README.rst [hunk 2]
```diff
@@ -28,6 +28,37 @@ Python 3.
...
 ``openscm``.
 
+Use guidelines
+--------------
+
+We encourage use of OpenSCM as much as possible and are open to collaboration.
+If you plan to publish using OpenSCM, please be respectful of the work and the `Maintainers`_' willingness to open source their code.
+
+In particular, when using OpenSCM, please cite the DOI of the precise version of the package used and consider citing our package description paper [when it's written, which it's not yet :)].
+As appropriate, please consider also citing the wrappers and models that OpenSCM relies.
```
Comment (swillner) [inline README.rst]:
```suggestion
As appropriate, please consider also citing the wrappers and models that OpenSCM relies on.
```

**Pull request merged**